src/Type/FloatType.php

```diff
@@ -239,6 +239,11 @@ public function isLowercaseString(): TrinaryLogic
 		return TrinaryLogic::createNo();
 	}
 
+	public function isUppercaseString(): TrinaryLogic
+	{
+		return TrinaryLogic::createNo();
```

Comment:
should be maybe - see https://github.com/phpstan/phpstan-src/pull/3613/files#r1835336748